test/e2e/util.go

```diff
@@ -316,6 +317,17 @@ var grafanaTestCases = []struct {
 			},
 		},
 	},
+	{
+		dashboardName: "network_topology_dashboard",
+		dashboardUid:  networkTopologyDashboardUid,
+		queryList: []query{
+			{
+				queryId:      0,
+				panelName:    "Network Topology",
+				expectResult: []string{controlPlaneNodeName(), workerNodeName(1), "perftest-a", "perftest-b", "perftest-c", "perftest-d", "perftest-e"},
```

Comment:
could you also add service names? as mentioned in: https://github.com/antrea-io/theia/pull/142#discussion_r1115077975